corehq/apps/locations/views.py

```diff
@@ -834,6 +834,16 @@ def location_export(request, domain):
     return response
 
 
+def get_locations_paylod_from_ids(ids, domain, base_queryset):
+    from corehq.apps.locations.util import get_locations_from_ids
+    locations = get_locations_from_ids(ids, domain, base_queryset=base_queryset)
+    return [loc_to_payload(loc) for loc in locations]
+
+
+def base_queryset_for_domain_and_user(domain, user):
+    return SQLLocation.objects.accessible_to_user(domain, user)
```

Comment:
Oh yes.. thats right. Admin would get access to locations in the domain only.
Ya! i thought i would give `self` a shot once i actually code this. Any filter would generally be after this so i didn't see chaining as a challenge then. But ya self!